daemon/cluster/executor/container/container.go

```diff
@@ -77,8 +77,18 @@ func (c *containerConfig) name() string {
 		return c.task.Annotations.Name
 	}
 
-	// fallback to service.slot.id.
-	return strings.Join([]string{c.task.ServiceAnnotations.Name, fmt.Sprint(c.task.Slot), c.task.ID}, ".")
+	// fallback to service.slot.id
+	return fmt.Sprintf("%s.%s", c.shortName(), c.task.ID)
```

Comment:
I was thinking that every task would get the following aliases:
* container id
* task name (service.slot)

(as well as the service name which is load balanced)